src/chain/member_info.rs

```diff
@@ -11,10 +11,15 @@
 pub struct AgeCounter(u32);
 
 impl AgeCounter {
+    #[cfg(any(test, feature = "mock_base"))]
     pub fn age(self) -> u8 {
         f64::from(self.0).log2() as u8
     }
 
+    pub fn set_age(&mut self, age: u8) {
```

Comment:
Should we be setting the age, or the age counter?

In relocations, the age should be enough, as they are supposed to happen when the age changes, so the age counter will be equal to some power of 2. But we might also want to set the counter directly when a node rejoins, for example - maybe it's the counter that we should care about internally, then?

Reply:
When we need that, then we should add `set_age_counter` or something to `MemberInfo`. Alternatively, we can have only `set_age_counter` and use something like `set_age_counter(AgeCounter::from_age(age))` instead of `set_age`. I suggest we make the change when we actually need it.